oaknational/Oak-Web-Application
# chore: change copy 'Back' to 'All blogs'

## Description

- change copy on blog post mobile filters

## Issue(s)

Fixes #804

## How to test

1. Go to {cloud link} on mobile
2. Click on filters
3. You should see "All blog" rather than "Back"


## Checklist
